omnibus/config/projects/chef.rb

```diff
@@ -51,15 +51,27 @@
 # Chef Release version pinning
 override :chef, version: "local_source"
 override :ohai, version: "master"
+override :"rb-readline", version: "v0.5.3"
+
+# Global FIPS override flag.
+if windows? || rhel?
+  override :fips, enabled: true
+end
```

Comment:
We think there isn't any.  The we ask ruby to build internal md5 but I don't anticipate anybody using chef to crack shitty encryption from the 90s and caring about raw md5 performance.